site/docs/concepts/config-policies.md

```diff
@@ -215,57 +215,45 @@ config[{"capability": "copy", "decision": decision}] {
 ```
 Note that an empty ConfigDecisions map will be returned if the expiration date is exceeded by the time when the policy is applied. 
 
-## Taking infrastructure metrics into consideration
 
-When writing configuration policies, infrastructure metrics and costs may also be taken into account in order to optimize the generated data plane. 
-For example, selection of a storage account may be based on a storage cost, selection of a cluster may provide a restriction on cluster capacity, and so on. 
-Infrastructure attributes and metrics are stored in the `/tmp/adminconfig/infrastructure.json` directory of the manager pod. Collection of the metrics and their dynamic update is beyond the scope of Fybrik. One may develop or use 3rd party solutions for monitoring and updating these infrastructure metrics.
+## Optimization goals
```

Comment:
We need to decide on terminology and be consistent throughout: are we calling these soft policies or optimization goals?  I'm fine with either as long as we're consistent in the code and in the documentation.

Reply:
I vote for optimization goals. I think that soft policies can be misleading inferring that any type of policy can be set to either, while we always refer to `config` as hard and `optimize` as soft. You can't, for example, prefer deployment of read in the workload cluster, it is either restricted or not.

Reply:
I'm fine with that as long as we're consistent.